enterprise/frontend/src/embedding-sdk/components/private/InteractiveAdHocQuestion.tsx

```diff
@@ -32,7 +32,6 @@ export const InteractiveAdHocQuestion = ({
       location={location}
       params={params}
       componentPlugins={plugins}
-      onReset={onNavigateBack}
```

Comment:
Do we have to do anything on the reset case?

Reply:
If `withResetButton` is `true`, the reset button will appear when a change is made to a question